libcontainer/integration/execin_test.go

```diff
@@ -30,16 +30,37 @@ func TestExecIn(t *testing.T) {
 	// Execute a first process in the container
 	stdinR, stdinW, err := os.Pipe()
 	ok(t, err)
+	defer func() {
+		_ = stdinR.Close()
+		_ = stdinW.Close()
+	}()
+	stdoutR, stdoutW, err := os.Pipe()
+	ok(t, err)
+	defer func() {
+		_ = stdoutR.Close()
+		_ = stdoutW.Close()
```

Comment:
can't we just defer the close, instead of a function? We are ignoring the error anyways.

I mean it for all the cases, not just this.